src/components/FeedbackForm/FeedbackForm.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import userEvent from "@testing-library/user-event"
+
+import { render, screen } from "../../utils/testUtils"
+import FeedbackForm from "./FeedbackForm"
+
+describe("FeedbackForm", () => {
```

Comment:
do we want to test the closing/clearing item metadata action?

Reply:
oh i forgot you said you haven't finished the testing in your comment. would love to see that in the future.

Reply:
will add this to the follow up testing ticket AC